bitrisescript/src/bitrisescript/bitrise.py

```diff
@@ -0,0 +1,218 @@
+import asyncio
+import logging
+import os
+from pprint import pformat
+from typing import Any
+
+from aiohttp_retry import RetryClient
+
+from scriptworker_client.exceptions import TaskVerificationError
+from scriptworker_client.utils import get_single_item_from_sequence
+
+BITRISE_API_URL = "https://api.bitrise.io/v0.1"
```

Comment:
As you've probably seen, [Bitrise says](https://devcenter.bitrise.io/en/api.html):
> There’s no long term compatibility promise for v0.1, although we try to do our best to not to break anything unless we have to. Once we’re happy with v0.1 we’ll “freeze” it as v1.0, for which we’ll provide long term support.

With this in mind, we may want to be cautious about building any release workflows on it for the time being.

Reply:
Hm, I don't know that we have much choice in that matter :/. Building release workflows is why we're triggering these from Taskcluster in the first place.

The good news is that we'll be using this for non-release workflows as well (e.g triggering builds + tests), so if something breaks, odds are it'll be on-push or pull-request instead of a release graph.